bin/kubectl-crossplane-stack-init

```diff
@@ -3,19 +3,28 @@
 set -e
 
 function usage {
-  echo "Usage: kubectl crossplane stack init STACK_NAME [DIRECTORY]" >&2
+  echo "Usage: kubectl crossplane stack init [-f] STACK_NAME [DIRECTORY]" >&2
   echo "" >&2
   echo "STACK_NAME is used as the image name for the stack, and any '/' characters" >&2
   echo "are converted to '-' characters when populating kubernetes resource field names." >&2
   echo "" >&2
   echo "DIRECTORY defaults to the current directory." >&2
+  echo "" >&2
+  echo "-f force init when a previous 'stack init' is detected"
```

Comment:
* This should be echoed to `stderr`
* Could you reword this to sound more like a sentence, like the other usage statements? For example, `-f/--force forces initialization, even when a previous . . . `